docs/v1/README.md

```diff
@@ -5,62 +5,67 @@ specification document.
 
 # Markup version
 
-The markup version uses Madoko (https://www.madoko.net) to produce
+The markup version uses AsciiDoc (https://docs.asciidoctor.org/) to produce
 HTML and PDF versions of the documentation. Pre-built versions of the
 documentation are available on the [P4.org specifications
 page](https://p4.org/specs).
 
 
 Files:
-- `P4Runtime-spec.mdk` is the main file. 
-- assets: Figures
-  - `*.odg` - OfficeLibre source drawing file used to export images. These are
+- `P4Runtime-Spec.adoc` is the main file. 
+- resources: 
+  - figs
+    - `*.odg` - OfficeLibre source drawing file used to export images. These are
     bulk-rendered at build time into .svg and .png images via `soffice`
     command-line (required in build environment)
+  - fonts 
+      - `*.ttf` - Type font source file used to export fonts.
+  - theme: 
+      - `*.yaml` - Describes how PDF P4Runtime specification will be displayed.
+      - `*.css`  - Describes how HTML P4Runtime specification will displayed.
+      - `*.bib`  - Bibliography file that contains a list of bibliographical item, such as articles, books, and theses.
 - `Makefile` builds documentation in the build subdirectory
-- `p4.json` is providing custom syntax highlighting for P4. It is a rip off from
-  the cpp.json provided by Madoko (the "extend" clause does not work, my version
-  of Madoko asks for a "tokenizer" to be defined). Style customization for each
-  token can be done using CSS style attributes (see `token.keyword` in
-  `P4Runtime-spec.mdk`).
+
 
 ## Building
 
-The easiest way to render the Madoko specification documentation is to use the
-`p4lang/p4rt-madoko:latest` Docker` image:
+The easiest way to render the AsciiDoc specification documentation is to use the
+`p4lang/p4rt-asciidoc:latest` Docker` image:
 
-    docker run -v `pwd`/docs/v1:/usr/src/p4-spec p4lang/p4rt-madoko:latest make
+    docker run -v `pwd`/docs/v1:/usr/src/p4-spec p4lang/p4rt-asciidoc:latest make
 
 ### Linux
 ```
-sudo apt-get install nodejs
-sudo npm install madoko -g
-sudo apt-get install libreoffice
-sudo apt-get install texlive-science texlive-xetex
-make [all | html | pdf ]
+rvm install ruby-3.3.1
```

Comment:
I would actually recommend omitting such steps here, in favor of just linking to the bash install script that you have already done.  I'd rather keep the correct instructions up to date in executable form in one place, rather than two places, in case they change in the future.  That reminds me that I should edit that script to install libreoffice, because the P4Runtime spec generation needs it.